lambda/s3_flush_lambda.py

```diff
@@ -89,7 +89,7 @@
 
         # Check if cuboid is black overwrite
         to_black = False 
-        if flush_msg_data["to_black"] == "True":
+        if flush_msg_data["to_black"] == "true":
```

Comment:
Sorry, didn't see the code you used in spdb to build this message. Once you make the change in there, you'll get a real boolean value, so then you can just use `if flush_msg_data["to_black"] == True:`. You still want to explicitly test for `True` since Python considers any value that's not `None` or `False` to be "true".